test/utils/page_objects/home_page/all_women_boots.js

```diff
@@ -0,0 +1,24 @@
+const HomePage = require("./home_page");
+const Element = require('../base_elements/base_element');
+
+
+class AllWomenBootsPage extends HomePage {
```

Comment:
This page extends `BasePage`, not `HomePage`.

Reply:
Done